Framework/PythonInterface/test/python/plugins/functions/PEARLTransVoigtTest.py

```diff
@@ -0,0 +1,22 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 20120 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+import unittest
+
+from MsdTestHelper import is_registered, do_a_fit
+from mantid.simpleapi import Fit
+
+
+class PEARLTransVoigtTest(unittest.TestCase):
+
+    def test_function_has_been_registered(self):
+        status, msg = is_registered("PEARLTransVoigt")
+        if not status:
+            self.fail(msg)
+
```

Comment:
So did Chris ever get back to you with any typical function parameters for the function? If no I see there are some default values in the fit function parameters. Please can you add a test to call function1D with some xvals that are in and around the default "position" of 1096.3 which I assume is the peak position in X and just assert in here that the current values are returned.
Will act as a regression test in future in case someone changes the function even if seems a bit circular right now